docassemble/juvenilesealing/data/questions/juvenile-sealing-text-10.yml

```diff
@@ -194,7 +192,9 @@ subquestion: |
   0 Jump St., Apt. # 2[BR]
   Boston, MA 02114
 
-continue button label: Keep going ${next_icon}
+buttons:
+  - I will come back to the quiz after ordering my docket sheets ${exit_icon}: True
```

Comment:
Not for this PR, but making a note to self: we should have a todo to tweak the wording here.

Reply:
Gotcha - I made an issue for this here: https://github.com/knod/docassemble-juvenilesealing/issues/51